bindings/python/tests/test_relation.py

```diff
@@ -0,0 +1,4 @@
+import dlite
+
+rel1 = dlite.Relation("s1", "p1", "o1")
+rel2 = dlite.Relation("s2", "p2", "o2", "d2")
```

Comment:
What are we testing here? Looks like we tests that setting a relation does not throw an error, but do we really test that it is there? Also, I would like tests on doing it wrongly as well, to check that the desired Errors are actually raised.

Reply:
Well spotted. Seems that I started on this test script, but forgot to finalise it. Added some more tests now.